Makefile

```diff
@@ -0,0 +1,28 @@
+TARGET:=$(GOPATH)/bin/grafana-reporter
+SRC:=$(GOPATH)/src/github.com/IzakMarais/reporter
+
+.PHONY: build
+build: $(TARGET)
+
+.PHONY: docker-build
+docker-build:
+	@docker build -t grafana-reporter:0.6.2 .
```

Comment:
Where does this version number 0.6.2. come from?

I should really start tagging releases and doing proper semver for this project....

Reply:
@IzakMarais was just using `version.go` https://github.com/IzakMarais/reporter/blob/master/cmd/grafana-reporter/version.go#L20-L22

Yeah, it'd be better if you used an `.env` file or something to read from that way build tools can get at the version easier.